pkg/apis/release/v1alpha1/types.go

```diff
@@ -144,20 +149,62 @@ type ReleasePayloadOverride struct {
 	Reason string `json:"reason,omitempty"`
 }
 
+type PayloadVerificationConfig struct {
+	// BlockingJobs are release verification jobs that will prevent a ReleasePayload from being Accepted if the job fails
+	BlockingJobs []CIConfiguration `json:"blockingJobs,omitempty"`
+	// InformingJobs are release verification jobs used to execute tests against a ReleasePayload
+	InformingJobs []CIConfiguration `json:"informingJobs,omitempty"`
+}
+
+// CIConfiguration is an Openshift CI system's job definition of a verification test to run against a ReleasePayload
+type CIConfiguration struct {
+	// CIConfigurationName the unique name given to a verification test.  This value will be used as the key to lookup
+	// the configuration and the results of the respective verification test
+	CIConfigurationName string `json:"name"`
+	// CIConfigurationJobName is the actual name of the prowjob definition as stored in the CI Job Configuration.  This
+	// value is used to lookup and read in the prowjob for processing by the release-controller
+	CIConfigurationJobName string `json:"jobName"`
+	// MaxRetries Maximum retry attempts for the job. Defaults to 0 - do not retry on fail
+	MaxRetries int `json:"maxRetries,omitempty"`
+	// UpgradeFrom Specifies an alternate upgrade source
+	UpgradeFrom CIConfigurationUpgradeFrom `json:"upgradeFrom,omitempty"`
```

Comment:
if you haven't used it, don't include it.

Reply:
Gone